index.html

```diff
@@ -66,19 +66,19 @@ <h3>I’am a Software Developer</h3>
     <ul class="social-icons">
       <li>
         <a rel="noopener noreferrer" href="https://github.com/Eckybrazzz" target="_blank"><img class="gitlogo"
-            src="assets/GitHub.svg" alt="GitHub" /></a>
+            src="assets/GitHub.svg" alt="My GitHub" /></a>
```

Comment:
- [ ] Also, here and in similar situations, I would suggest updating these values so the users know that this is an icon :raised_hands: :pushpin: